test/mysql-config/my.cnf

```diff
@@ -0,0 +1,2 @@
+[mysqld]
+sql_mode = "NO_AUTO_CREATE_USER,NO_ENGINE_SUBSTITUTION"
```

Comment:
can you put a comment here explaining what this does and why it's needed for the tests?